### PR TITLE
ArgumentsAnalyzer - pass by reference not analyzed correctly

### PR DESCRIPTION
**Pull request by SpacePossum**
name ended up being `array&`

### CODE REVIEW

File: tests/AutoReview/FixerTest.php [hunk 1]
```diff
@@ -186,6 +187,66 @@ public function testFixersAreFinal(FixerInterface $fixer)
...
+    /**
+     * @dataProvider provideFixerDefinitionsCases
+     */
+    public function testFixersHandleEmptyTokens(FixerInterface $fixer)
+    {
+        $tokens = Tokens::fromCode('');
+
+        try {
+            $fixer->fix(new \SplFileInfo(__FILE__), $tokens);
+        } catch (RequiredFixerConfigurationException $e) {
```
Comment (keradus) [inline tests/AutoReview/FixerTest.php]: if this happen, we do not know that fixer works for empty file

by silencing the error, we lied to our-self that it does

File: tests/Tokenizer/Analyzer/ArgumentsAnalyzerTest.php [hunk 2]
```diff
@@ -126,6 +126,16 @@ public function provideArgumentsInfoCases()
                     3
                 )
             )],
+            ['<?php function(array & $a){};', 3, 7, new ArgumentAnalysis(
```
Comment (keradus) [inline tests/Tokenizer/Analyzer/ArgumentsAnalyzerTest.php]: :+1:

**Comment by dmvdbrugge** [comment]
You could've save the trouble, as this was already in pr: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3924 ...

**Comment by keradus** [comment]
@SpacePossum , can you please leave `testFixersHandleSingleToken` (and similar) tests here only and merge 3924 first ?

**Comment by keradus** [comment]
@SpacePossum , I merged 3924, please rebase to leave only tests for empty content

**Comment by SpacePossum** [comment]
nothing much of value here, closing as such

**Pull request closed without merging**